lib/ladybug/energy_model.rb

```diff
@@ -29,41 +29,41 @@
 # OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 # *******************************************************************************
 
-require 'ladybug/energy_model/version'
-require 'ladybug/energy_model/extension'
-require 'ladybug/energy_model/extension_simulation_parameter'
-require 'ladybug/energy_model/aperture'
-require 'ladybug/energy_model/energy_material'
-require 'ladybug/energy_model/energy_material_no_mass'
-require 'ladybug/energy_model/energy_window_material_gas'
-require 'ladybug/energy_model/energy_window_material_gas_mixture'
-require 'ladybug/energy_model/energy_window_material_gas_custom'
-require 'ladybug/energy_model/energy_window_material_blind'
-require 'ladybug/energy_model/energy_window_material_glazing'
-require 'ladybug/energy_model/energy_window_material_shade'
-require 'ladybug/energy_model/energy_window_material_simpleglazsys'
-require 'ladybug/energy_model/opaque_construction_abridged'
-require 'ladybug/energy_model/window_construction_abridged'
-require 'ladybug/energy_model/shade_construction'
-require 'ladybug/energy_model/construction_set'
-require 'ladybug/energy_model/face'
-require 'ladybug/energy_model/model'
-require 'ladybug/energy_model/model_object'
-require 'ladybug/energy_model/room'
-require 'ladybug/energy_model/aperture'
-require 'ladybug/energy_model/door'
-require 'ladybug/energy_model/shade'
-require 'ladybug/energy_model/schedule_type_limit'
-require 'ladybug/energy_model/schedule_fixed_interval_abridged'
-require 'ladybug/energy_model/schedule_ruleset_abridged'
-require 'ladybug/energy_model/space_type'
-require 'ladybug/energy_model/people_abridged'
-require 'ladybug/energy_model/lighting_abridged'
-require 'ladybug/energy_model/electric_equipment_abridged'
-require 'ladybug/energy_model/gas_equipment_abridged'
-require 'ladybug/energy_model/infiltration_abridged'
-require 'ladybug/energy_model/ventilation_abridged'
-require 'ladybug/energy_model/setpoint_thermostat'
-require 'ladybug/energy_model/setpoint_humidistat'
-require 'ladybug/energy_model/ideal_air_system'
-require 'ladybug/energy_model/simulation_parameter'
\ No newline at end of file
+require "#{File.dirname(__FILE__)}/version"
```

Comment:
@chriswmackey As per the folder structure, the address for this should be require `"#{File.dirname(__FILE__)}/energy_model/version"` .